koku/masu/processor/tasks.py

```diff
@@ -498,6 +498,10 @@ def update_summary_tables(  # noqa: C901
         Provider.PROVIDER_AWS_LOCAL,
         Provider.PROVIDER_AZURE,
         Provider.PROVIDER_AZURE_LOCAL,
+        Provider.PROVIDER_GCP,
+        Provider.PROVIDER_GCP_LOCAL,
+        Provider.PROVIDER_OCI,
+        Provider.PROVIDER_OCI_LOCAL,
     ):
```

Comment:
Might be easier to just swap this to say If the provider isn't OCP. Would prevent us from having to update it in the future if we add any extra providers down the line

Reply:
Thanks @cgoodfred, its cleaner this way.